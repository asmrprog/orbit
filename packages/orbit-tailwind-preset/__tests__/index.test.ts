import orbitPreset from "../src";

describe("foundationPreset", () => {
  it("should match snapshot", () => {
    expect(orbitPreset({ disablePreflight: true, includeComponentTokens: true })).toMatchSnapshot();
  });
});
