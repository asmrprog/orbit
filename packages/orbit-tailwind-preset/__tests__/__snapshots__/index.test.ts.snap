// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`foundationPreset should match snapshot 1`] = `
Object {
  "corePlugins": Object {
    "preflight": false,
  },
  "prefix": "orbit-",
  "presets": Array [
    Object {
      "theme": Object {
        "borderRadius": Object {
          "circle": "50%",
          "large": "6px",
          "normal": "3px",
          "small": "2px",
        },
        "boxShadow": Object {
          "action": "0 0 2px 0 rgba(37, 42, 49, 0.16),0 1px 4px 0 rgba(37, 42, 49, 0.12)",
          "action-active": "0 1px 4px 0 rgba(37, 42, 49, 0.16),0 4px 8px 0 rgba(37, 42, 49, 0.12)",
          "fixed": "0 0 2px 0 rgba(37, 42, 49, 0.16),0 2px 4px 0 rgba(37, 42, 49, 0.12)",
          "fixed-reverse": "0 0 2px 0 rgba(37, 42, 49, 0.16),0 -2px 4px 0 rgba(37, 42, 49, 0.12)",
          "none": "none",
          "overlay": "0 12px 24px -4px rgba(37, 42, 49, 0.24),0 8px 60px 0 rgba(37, 42, 49, 0.32)",
          "raised": "0 4px 8px 0 rgba(37, 42, 49, 0.16),0 8px 24px 0 rgba(37, 42, 49, 0.24)",
          "raised-reverse": "0 -4px 8px 0 rgba(37, 42, 49, 0.16),0 -8px 24px 0 rgba(37, 42, 49, 0.24)",
        },
        "colors": Object {
          "blue": Object {
            "dark": "#005AA3",
            "dark-active": "#003E70",
            "dark-hover": "#004F8F",
            "darker": "#004680",
            "light": "#E8F4FD",
            "light-active": "#C7E4FA",
            "light-hover": "#DEF0FC",
            "normal": "#0172CB",
            "normal-active": "#01508E",
            "normal-hover": "#0161AC",
          },
          "cloud": Object {
            "dark": "#BAC7D5",
            "dark-active": "#94A8BE",
            "dark-hover": "#A6B6C8",
            "light": "#F5F7F9",
            "light-active": "#D6DEE6",
            "light-hover": "#E5EAEF",
            "normal": "#E8EDF1",
            "normal-active": "#CAD4DE",
            "normal-hover": "#DCE3E9",
          },
          "green": Object {
            "dark": "#2D7738",
            "dark-active": "#1F5126",
            "dark-hover": "#276831",
            "darker": "#235C2B",
            "light": "#EAF5EA",
            "light-active": "#CDE4CF",
            "light-hover": "#E1EFE2",
            "normal": "#28A138",
            "normal-active": "#1D7228",
            "normal-hover": "#238B31",
          },
          "ink": Object {
            "dark": "#252A31",
            "dark-active": "#0B0C0F",
            "dark-hover": "#181B20",
            "light": "#697D95",
            "light-active": "#4A617C",
            "light-hover": "#5D738E",
            "normal": "#4F5E71",
            "normal-active": "#324256",
            "normal-hover": "#3E4E63",
          },
          "orange": Object {
            "dark": "#AD5700",
            "dark-active": "#954A00",
            "dark-hover": "#A75400",
            "darker": "#803F00",
            "light": "#FEF2E6",
            "light-active": "#FAE2C6",
            "light-hover": "#FCECDA",
            "normal": "#DF7B00",
            "normal-active": "#B26200",
            "normal-hover": "#C96F00",
          },
          "product": Object {
            "dark": "#007A69",
            "dark-active": "#006657",
            "dark-hover": "#007060",
            "darker": "#005C4E",
            "light": "#E1F4F3",
            "light-active": "#BFE8E2",
            "light-hover": "#D6F0EC",
            "normal": "#00A58E",
            "normal-active": "#008472",
            "normal-hover": "#009580",
          },
          "red": Object {
            "dark": "#970C0C",
            "dark-active": "#6D0909",
            "dark-hover": "#890B0B",
            "darker": "#760909",
            "light": "#FAEAEA",
            "light-active": "#F3CECE",
            "light-hover": "#F8E2E2",
            "normal": "#D21C1C",
            "normal-active": "#9D1515",
            "normal-hover": "#B91919",
          },
          "social": Object {
            "facebook": "#3B5998",
            "facebook-active": "#354F88",
            "facebook-hover": "#385490",
          },
          "white": Object {
            "normal": "#FFFFFF",
            "normal-active": "#E7ECF1",
            "normal-hover": "#F1F4F7",
          },
        },
        "fontFamily": Object {
          "base": "'Roboto', -apple-system, '.SFNSText-Regular', 'San Francisco', 'Segoe UI', 'Helvetica Neue', 'Lucida Grande', sans-serif",
        },
        "fontWeight": Object {
          "bold": "700",
          "medium": "500",
          "normal": "400",
        },
        "lineHeight": Object {
          "large": "24px",
          "normal": "20px",
          "small": "16px",
        },
        "screens": Object {
          "de": 992,
          "ld": 1200,
          "lm": 576,
          "mm": 414,
          "sm": "320px",
          "tb": 768,
        },
        "spacings": Object {
          "0": "0px",
          "lg": undefined,
          "md": undefined,
          "none": "0px",
          "sm": undefined,
          "xl": undefined,
          "xs": undefined,
          "xxl": undefined,
          "xxs": undefined,
          "xxxl": undefined,
          "xxxs": undefined,
        },
        "transitionDuration": Object {
          "fast": "0.15s",
          "normal": "0.3s",
          "slow": "0.4s",
        },
      },
    },
    Object {
      "theme": Object {
        "extend": Object {
          "colors": Object {
            "badge-bundle-basic-background": "linear-gradient(to top right, #E13E3B 0%, #E87E09 100%)",
            "badge-bundle-basic-foreground": "#FFFFFF",
            "badge-bundle-medium-background": "linear-gradient(to top right, #3719AB 0%, #8539DB 100%)",
            "badge-bundle-medium-foreground": "#FFFFFF",
            "badge-bundle-top-background": "linear-gradient(to top right, #2D2D2E 0%, #696E73 100%)",
            "badge-bundle-top-foreground": "#FFFFFF",
            "badge-critical-background": "#D21C1C",
            "badge-critical-foreground": "#FFFFFF",
            "badge-critical-subtle-background": "#FAEAEA",
            "badge-critical-subtle-foreground": "#970C0C",
            "badge-dark-background": "#252A31",
            "badge-dark-foreground": "#FFFFFF",
            "badge-info-background": "#0172CB",
            "badge-info-foreground": "#FFFFFF",
            "badge-info-subtle-background": "#E8F4FD",
            "badge-info-subtle-foreground": "#005AA3",
            "badge-neutral-background": "#F5F7F9",
            "badge-neutral-foreground": "#252A31",
            "badge-success-background": "#28A138",
            "badge-success-foreground": "#FFFFFF",
            "badge-success-subtle-background": "#EAF5EA",
            "badge-success-subtle-foreground": "#2D7738",
            "badge-warning-background": "#DF7B00",
            "badge-warning-foreground": "#FFFFFF",
            "badge-warning-subtle-background": "#FEF2E6",
            "badge-warning-subtle-foreground": "#AD5700",
            "badge-white-background": "#FFFFFF",
            "badge-white-foreground": "#252A31",
            "button-bundle-basic-background": "linear-gradient(to top right, #E13E3B 0%, #E87E09 100%)",
            "button-bundle-basic-background-active": "linear-gradient(to top right, #9F1816 0%, #C36802 100%)",
            "button-bundle-basic-background-hover": "linear-gradient(to top right, #BD2825 0%, #D67000 100%)",
            "button-bundle-medium-background": "linear-gradient(to top right, #3719AB 0%, #8539DB 100%)",
            "button-bundle-medium-background-active": "linear-gradient(to top right, #250F79 0%, #5A3485 100%)",
            "button-bundle-medium-background-hover": "linear-gradient(to top right, #BD2825 0%, #D67000 100%)",
            "button-bundle-top-background": "linear-gradient(to top right, #2D2D2E 0%, #696E73 100%)",
            "button-bundle-top-background-active": "linear-gradient(to top right, #101011 0%, #51575C)",
            "button-bundle-top-background-hover": "linear-gradient(to top right, #171718 0%, #51575C 100%)",
            "button-critical-background": "#D21C1C",
            "button-critical-background-active": "#9D1515",
            "button-critical-background-hover": "#B91919",
            "button-critical-foreground": "#FFFFFF",
            "button-critical-foreground-active": "#FFFFFF",
            "button-critical-foreground-hover": "#FFFFFF",
            "button-critical-subtle-background": "#FAEAEA",
            "button-critical-subtle-background-active": "#F3CECE",
            "button-critical-subtle-background-hover": "#F8E2E2",
            "button-critical-subtle-foreground": "#970C0C",
            "button-critical-subtle-foreground-active": "#6D0909",
            "button-critical-subtle-foreground-hover": "#890B0B",
            "button-info-background": "#0172CB",
            "button-info-background-active": "#01508E",
            "button-info-background-hover": "#0161AC",
            "button-info-foreground": "#FFFFFF",
            "button-info-foreground-active": "#FFFFFF",
            "button-info-foreground-hover": "#FFFFFF",
            "button-link-critical-background": "transparent",
            "button-link-critical-background-active": "#F3CECE",
            "button-link-critical-background-hover": "#F8E2E2",
            "button-link-critical-foreground": "#D21C1C",
            "button-link-critical-foreground-active": "#9D1515",
            "button-link-critical-foreground-hover": "#B91919",
            "button-link-primary-background": "transparent",
            "button-link-primary-background-active": "#BFE8E2",
            "button-link-primary-background-hover": "#D6F0EC",
            "button-link-primary-foreground": "#00A58E",
            "button-link-primary-foreground-active": "#006657",
            "button-link-primary-foreground-hover": "#007060",
            "button-link-secondary-background": "transparent",
            "button-link-secondary-background-active": "#D6DEE6",
            "button-link-secondary-background-hover": "#E5EAEF",
            "button-link-secondary-foreground": "#252A31",
            "button-link-secondary-foreground-active": "#0B0C0F",
            "button-link-secondary-foreground-hover": "#181B20",
            "button-primary-background": "#00A58E",
            "button-primary-background-active": "#008472",
            "button-primary-background-hover": "#009580",
            "button-primary-foreground": "#FFFFFF",
            "button-primary-foreground-active": "#FFFFFF",
            "button-primary-foreground-hover": "#FFFFFF",
            "button-primary-subtle-background": "#E1F4F3",
            "button-primary-subtle-background-active": "#BFE8E2",
            "button-primary-subtle-background-hover": "#D6F0EC",
            "button-primary-subtle-foreground": "#007A69",
            "button-primary-subtle-foreground-active": "#006657",
            "button-primary-subtle-foreground-hover": "#007060",
            "button-secondary-background": "#E8EDF1",
            "button-secondary-background-active": "#CAD4DE",
            "button-secondary-background-hover": "#DCE3E9",
            "button-secondary-foreground": "#252A31",
            "button-secondary-foreground-active": "#0B0C0F",
            "button-secondary-foreground-hover": "#181B20",
            "button-success-background": "#28A138",
            "button-success-background-active": "#1D7228",
            "button-success-background-hover": "#238B31",
            "button-success-foreground": "#FFFFFF",
            "button-success-foreground-active": "#FFFFFF",
            "button-success-foreground-hover": "#FFFFFF",
            "button-warning-background": "#DF7B00",
            "button-warning-background-active": "#B26200",
            "button-warning-background-hover": "#C96F00",
            "button-warning-foreground": "#FFFFFF",
            "button-warning-foreground-active": "#FFFFFF",
            "button-warning-foreground-hover": "#FFFFFF",
            "button-white-background": "#FFFFFF",
            "button-white-background-active": "#E5EAEF",
            "button-white-background-hover": "#F5F7F9",
            "button-white-foreground": "#252A31",
            "button-white-foreground-active": "#0B0C0F",
            "button-white-foreground-hover": "#181B20",
            "form-element-background": "#FFFFFF",
            "form-element-disabled-background": "#E8EDF1",
            "form-element-disabled-foreground": "#BAC7D5",
            "form-element-filled-foreground": "#4F5E71",
            "form-element-foreground": "#BAC7D5",
            "form-element-label-filled-foreground": "#697D95",
            "form-element-label-foreground": "#4F5E71",
            "form-element-prefix-foreground": "#697D95",
            "heading-foreground": "#252A31",
            "tag-colored-background": "#E8F4FD",
            "tag-colored-background-active": "#C7E4FA",
            "tag-colored-background-hover": "#DEF0FC",
            "tag-colored-foreground": "#004680",
            "tag-neutral-background": "#E8EDF1",
            "tag-neutral-background-active": "#CAD4DE",
            "tag-neutral-background-hover": "#DCE3E9",
            "tag-neutral-foreground": "#252A31",
            "text-critical-foreground": "#D21C1C",
            "text-info-foreground": "#0172CB",
            "text-link-foreground-hover": "#009580",
            "text-link-primary-foreground": "#007A69",
            "text-link-secondary-foreground": "#252A31",
            "text-primary-foreground": "#252A31",
            "text-secondary-foreground": "#4F5E71",
            "text-success-foreground": "#28A138",
            "text-warning-foreground": "#DF7B00",
            "text-white-foreground": "#FFFFFF",
          },
          "fontSize": Object {
            "button-large-font-size": "16px",
            "button-normal-font-size": "15px",
            "button-small-font-size": "13px",
            "form-element-normal-font-size": "15px",
            "form-element-small-font-size": "15px",
            "heading-display-font-size": "40px",
            "heading-display-subtitle-font-size": "22px",
            "heading-title1font-size": "28px",
            "heading-title2font-size": "22px",
            "heading-title3font-size": "16px",
            "heading-title4font-size": "15px",
            "heading-title5font-size": "13px",
            "heading-title6font-size": "13px",
          },
          "fontWeight": Object {
            "heading-display-font-weight": "700",
            "heading-display-subtitle-font-weight": "400",
            "heading-title1font-weight": "700",
            "heading-title2font-weight": "500",
            "heading-title3font-weight": "500",
            "heading-title4font-weight": "500",
            "heading-title5font-weight": "700",
            "heading-title6font-weight": "700",
            "text-link-font-weight": "500",
          },
          "lineHeight": Object {
            "heading-display-line-height": "44px",
            "heading-display-subtitle-line-height": "28px",
            "heading-title1line-height": "36px",
            "heading-title2line-height": "28px",
            "heading-title3line-height": "24px",
            "heading-title4line-height": "20px",
            "heading-title5line-height": "20px",
            "heading-title6line-height": "20px",
          },
          "zIndex": Object {
            "default": 1,
            "drawer": 815,
            "modal": 825,
            "modal-overlay": 800,
            "overlay": 800,
            "sticky": 100,
          },
        },
      },
    },
  ],
}
`;
